### PR TITLE
Make migration path configurable

### PR DESCRIPTION
**Pull request by theofidry**
In https://github.com/theofidry/AliceDataFixtures I also need access to this migration path so making it configurable via a parameter makes it easier :)

### CODE REVIEW

File: resources/config/services.xml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" ?>
 <container xmlns="http://symfony.com/schema/dic/services">
 
+    <parameters>
+        <parameter key="wouterj_eloquent.migration_path" type="string">%kernel.root_dir%/migrations</parameter>
```
Comment (wouterj) [inline resources/config/services.xml]: you should remove `type="string"`. `%kernel.root_dir%` should be resolved.

**Comment by wouterj** [comment]
There are quite some test failures, can you please take a look at those?

**Comment by theofidry** [comment]
I'll do tomorrow, I believe it's because you're using a Test command class which relies on the container, but the kernel is not booted so the parameter is not found

**Comment by wouterj** [comment]
ping @theofidry 

I'm planning on releasing a new minor release this weekend. If you can fix this before that, it would be awesome. Otherwise, I'll try to finish this on saturday.

**Comment by theofidry** [comment]
@wouterj soz will try to have a look tomorrow depending of how it goes at work. I won't be able to finish it this weekend so it will either be tomorrow or nothing

**Comment by wouterj** [comment]
No problem. I'm happy enough to have someone submitting PRs for this bundle :)

Finished this in https://github.com/wouterj/WouterJEloquentBundle/pull/28

**Pull request closed without merging**